specification/components/r4/schemas/responses/retrieveBinary/307Response.yaml

```diff
@@ -0,0 +1,12 @@
+description: Response redirects user agent to the large file attachment URL in the Object Store via the Location header.
+headers:
+  x-correlation-id:
+    $ref: '../../headers/response/CorrelationID.yaml'
+  x-request-id:
+    $ref: '../../headers/response/RequestID.yaml'
+  Location:
+    description: The attachment location in the object store.
+    required: true
+    schema:
+      type: string
+      example: 'TODO Do we want to provide an example?'
```

Comment:
I wonder if it's simpler to not include an example here just so clients to make any assumptions about the URLs format?